interfaces/Portal/src/app/program/edit-person-affected-popup/edit-person-affected-popup.component.ts

```diff
@@ -142,15 +142,16 @@ export class EditPersonAffectedPopupComponent implements OnInit {
     attribute: string,
     value: string | number | string[],
     reason: string,
-    isPaTableAttribute: boolean,
+    isProgramRegistrationAttribute: boolean,
   ): Promise<void> {
     let valueToStore: string | number | string[];
 
     valueToStore = value;
 
-    if (isPaTableAttribute && !Array.isArray(value)) {
-      valueToStore = String(value);
+    if (isProgramRegistrationAttribute && value === '') {
```

Comment:
are we losing functionality here were a number was converted to string?

Reply:
yes but the backend handles this: 
![image](https://github.com/user-attachments/assets/0812854a-d304-4844-9184-dcfdc21994dc)